pilot/helpers/Project.py

```diff
@@ -91,10 +90,59 @@ def __init__(
         if os.getenv("AUTOFIX_FILE_PATHS", "").lower() in ["true", "1", "yes"]:
             File.update_paths()
 
+        # start loading of project (since backwards compatibility)
+        self.should_overwrite_files = False
+        self.last_detailed_user_review_goal = None
+        self.last_iteration = None
+        self.tasks_to_load = []
+        self.features_to_load = []
+        self.dev_steps_to_load = []
+        if self.continuing_project:
+            self.setup_loading()
+        # end loading of project
+
     def set_root_path(self, root_path: str):
         self.root_path = root_path
         self.dot_pilot_gpt.with_root_path(root_path)
 
+    def setup_loading(self):
```

Comment:
This function is not just copy paste of previous logic. It works differently so try to understand it and keep an eye to details 👀